test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -386,27 +392,38 @@ func TestFeatureToggles(t *testing.T) {
 				// Verify that the CRB is back
 				_, err = wait.For(t, memberAwait.Awaitility, &v1.ClusterRoleBinding{}).WithNameThat(crbName)
 				require.NoError(t, err)
+				// the noise CRB for unknown/disabled feature is still not created
+				err = wait.For(t, memberAwait.Awaitility, &v1.ClusterRoleBinding{}).WithNameDeleted(noiseCrbName)
+				require.NoError(t, err)
 			})
 		})
 	})
 }
 
-func withClusterRoleBinding(t *testing.T, otherTier *toolchainv1alpha1.NSTemplateTier, feature string) tiers.CustomNSTemplateTierModifier {
-	var tpl bytes.Buffer
-	err := template.Must(template.New("crb").Parse(viewCRB)).Execute(&tpl, map[string]interface{}{
-		"featureName": feature,
-	})
-	require.NoError(t, err)
+var unknownFeature = "unknown-feature"
+
+func withClusterRoleBindings(t *testing.T, otherTier *toolchainv1alpha1.NSTemplateTier, feature string) tiers.CustomNSTemplateTierModifier {
+	clusterRB := getCRBforFeature(t, feature)       // This is the ClusterRoleBinding for the desired feature
+	noiseCRB := getCRBforFeature(t, unknownFeature) // This is a noise CRB for unknown/disabled feature. To be used to check that this CRB is never created.
 
 	return tiers.WithClusterResources(t, otherTier, func(template *toolchainv1alpha1.TierTemplate) error {
-		clusterRB := runtime.RawExtension{
-			Raw: tpl.Bytes(),
-		}
-		template.Spec.Template.Objects = append(template.Spec.Template.Objects, clusterRB)
+		template.Spec.Template.Objects = append(template.Spec.Template.Objects, clusterRB, noiseCRB)
 		return nil
 	})
 }
 
+func getCRBforFeature(t *testing.T, featureName string) runtime.RawExtension {
+	var enabledFeature bytes.Buffer
```

Comment:
minor - should we rename this to something else? in case of the unknown-feature it's not enabledFeature.

Reply:
Good catch. That name was a leftover. Renamed here: https://github.com/codeready-toolchain/toolchain-e2e/pull/1016/commits/79b82259425af05d3f1ddccb683f1eede5199bcb